awx/ui_next/src/screens/Host/Hosts.jsx

```diff
@@ -64,6 +67,7 @@ class Hosts extends Component {
                     location={location}
                     setBreadcrumb={this.setBreadcrumbConfig}
                     me={me || {}}
+                    inventory={inventory}
```

Comment:
When would `inventory` have a value here?

Reply:
It doesn't. Removed

Reply:
It doesn't.  Removed